bitbots_body_behavior/bitbots_body_behavior/decisions/reached_goal.py

```diff
@@ -40,23 +40,22 @@ def get_reevaluate(self):
         return True
 
 
-class AlignedToMoveBaseGoal(AbstractDecisionElement):
+class AlignedToGoal(AbstractDecisionElement):
     blackboard: BodyBlackboard
     def __init__(self, blackboard, dsd, parameters=None):
-        super(AlignedToMoveBaseGoal, self).__init__(blackboard, dsd, parameters)
+        super(AlignedToGoal, self).__init__(blackboard, dsd, parameters)
         self.orientation_threshold = self.blackboard.config['goal_alignment_orientation_threshold']  # [deg]
 
     def perform(self, reevaluate=False):
         """
-        It is determined if the robot is correctly aligned to the orientation of the move_base goal within a
+        It is determined if the robot is correctly aligned to the orientation of the goal within a
         determined threshold by comparing the current orientation angle of the robot in the map with the one from the
-        move_base goal.
+        goal.
         """
         current_pose = self.blackboard.world_model.get_current_position_pose_stamped()
         current_goal = self.blackboard.pathfinding.get_goal()
         if current_pose is None or current_goal is None:
-            # When move_base did not received a goal yet, no current position on the map is known.
```

Comment:
I think this part of the comment is still needed. It just describes the default behavior if we have no path planning goal -> we are not aligned to it.